src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -16,6 +16,7 @@
 
 package com.spotify.folsom.client;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
I think usage of `VisibleForTesting` is usually a sign of design/layering problems.

Expose a metric instead?